rewrite-pattern.js

```diff
@@ -363,21 +462,34 @@ const rewritePattern = (pattern, flags, options) => {
 	validateOptions(options);
 
 	config.flags.unicode = hasFlag(flags, 'u');
+	config.flags.unicodeSet = hasFlag(flags, 'v');
 	config.flags.ignoreCase = hasFlag(flags, 'i');
 	config.flags.dotAll = hasFlag(flags, 's');
 
 	config.transform.dotAllFlag = config.flags.dotAll && transform(options, 'dotAllFlag');
-	config.transform.unicodeFlag = config.flags.unicode && transform(options, 'unicodeFlag');
+	config.transform.unicodeFlag = (config.flags.unicode || config.flags.unicodeSet) && transform(options, 'unicodeFlag');
+	config.transform.unicodeSetFlag = config.flags.unicodeSet && transform(options, 'unicodeSetFlag');
+
 	// unicodeFlag: 'transform' implies unicodePropertyEscapes: 'transform'
 	config.transform.unicodePropertyEscapes = config.flags.unicode && (
 		transform(options, 'unicodeFlag') || transform(options, 'unicodePropertyEscapes')
 	);
 	config.transform.namedGroups = transform(options, 'namedGroups');
 
+	const regjsparserFeatures = {
+		'unicodeSet': Boolean(options && options.unicodeSetFlag),
```

Comment:
Perhaps `unicodeSets` here too, for consistency?

Reply:
This is a regjsparser option, so we could change it when it has a breaking release. However, it also uses `unicodePropertyEscape` so it's consistent with itself.